src/pages/FillBlankModel.css

```diff
@@ -0,0 +1,49 @@
+.fill-title {
+    font-size: 15px;
+    margin-top:50px;
+    font-family: sans-serif;
```

Comment:
You don't have to set the font type here, if you don't set one it will use the default one which is the correct one